### ISSUE TITLE
[Question] How does the cursor based incremental export work?

### ISSUE THREAD

**Issue by nathanvj** [justeat/ZendeskApiClient#266]
I'm using version `4.0.1-alpha-696` and I'm wondering when I call the following method e.g.:

```Task<UsersListCursorResponse> GetAllAsync(CursorPager pager, CancellationToken cancellationToken = default(CancellationToken));```

The `CursorPager` object only has one field, named the `Size`. How do I:
a. Specify the start unix datetime to use for the initial request
b. Specify the cursor to use

Thanks in advance.

**Comment by fbvilela** [comment]
@nathanvj how did you go with this one? I'm trying to find answers as well on how to loop through all the pages from a cursor paginated response, I'm guessing that's what you wanted when you mentioned `b. Specify the cursor` 
thanks

**Comment by nathanvj** [comment]
@fbvilela You're right, that's what I wanted to achieve. I couldn't find a way to do it with this package, so I build my own API interface using [Refit](https://github.com/reactiveui/refit).

**Comment by mikerogers123** [comment]
Hi @nathanvj you should be able to achieve this now, as of release [7.0.1](https://github.com/justeat/ZendeskApiClient/releases/tag/v7.0.1).
